OptimizelySDK/Config/OptimizelyConfigHandler.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Configuration;
```

Comment:
Needs license header. Also, can you add a doc description for what this class is for? 

Also, if this also has config options for the event processor, it should not be in the config package. Instead it should be on the same level as the client. We don't need to have `Optimizely` in the name. Instead it can be `ClientConfigHandler`